spec/controllers/settings/profiles_controller_spec.rb

```diff
@@ -23,5 +23,10 @@
       expect(account.reload.display_name).to eq 'New name'
       expect(response).to redirect_to(settings_profile_path)
     end
+
+    it 'renders :show if parameter is missing' do
+      put :update
+      expect(response).to render_template :show
```

Comment:
> Return a 400 as an error because the params are not valid when it's missing, or;

Here I'm considering the inconsistency with validation failure in `account.update` is problematic. (I would not like to spec that.) I think it is more convincing to return 400 for validation failure as well if it returns 400 for missing parameter, isn't it?